src/User32.Desktop/User32.cs

```diff
@@ -88,7 +88,7 @@ public static partial class User32
             int nMaxCount);
 
         [DllImport(nameof(User32), SetLastError = true)]
-        public static extern uint GetWindowThreadProcessId(IntPtr hWnd, out int lpdwProcessId);
+        public static extern int GetWindowThreadProcessId(IntPtr hWnd, out int lpdwProcessId);
```

Comment:
Although it's a breaking change at this point. But I guess it's a brand newly added method from your last PR, so we're good to go here. :+1: